_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -992,7 +992,8 @@ void FillPWT(
         mfxExtCodingOption2   & extOpt2     = GetExtBufferRef(par);
         mfxExtFeiSliceHeader  * extFeiSlice = GetExtBuffer(par, idxToPickBuffer);
         assert(extDdi      != 0);
-        assert(extFeiSlice != 0);
+        if (extFeiSlice == nullptr)
+            throw std::logic_error(": no external buffer found");
```

Comment:
Worth to add to string "ExtFeiSliceHeader"

Reply:
Actually there are always 2 buffers of mfxExtFeiSliceHeader present in MfxVideoParam, so you can use GetExtBufRef here as well and to not introduce exceptions

Reply:
we have not overload GetExtBufferRef(par, **field**)

Reply:
ok, so I would prefer if we have such overload instead of introducing exceptions here